study.md

```diff
@@ -46,7 +73,11 @@ other named Mr.MagicNibbles (yellow) and bites a bit. Please copy the JSON you
 already wrote and include the information about Dave's fish.
 
 ```json
-<!-- your answer here -->
+"fishDescription" : {
```

Comment:
this really isn't specific enough, what if i wanted to find the color of one of the fish, plus what about the one that bites alot?